security/libu2f-host/files/patch-u2fmisc-macro-fix.diff

```diff
@@ -0,0 +1,16 @@
+--- u2f-host/u2fmisc.c.orig	2019-02-22 03:31:45.000000000 -0800
++++ u2f-host/u2fmisc.c	2021-08-12 19:08:00.000000000 -0700
+@@ -33,6 +33,13 @@
+ #define u2fh_json_object_object_get(obj, key, value) (value = json_object_object_get(obj, key)) == NULL ? (json_bool)FALSE : (json_bool)TRUE
+ #endif
+ 
++#ifndef FALSE
++#define FALSE 0
++#endif
++#ifndef TRUE
++#define TRUE  1
++#endif
```

Comment:
Please use upstream patch for this PR as well. https://github.com/Yubico/libu2f-host/commit/840f01135d2892f45e71b9e90405de587991bd03

Reply:
I was working on that after the libu2f-server update. Thank you.